### PR TITLE
Lifecycle Update: OTel semantic conventions.

### PR DESCRIPTION
**Pull request by brett0000FF**
### What does this PR do? What is the motivation?

Updating all docs to reflect [product lifecycle changes](https://datadoghq.atlassian.net/wiki/spaces/PM/pages/3310617880/Updated+Product+Lifecycle+DatadogunreadCommentMark=67c51abe-864c-419a-90ef-79e6e4934f84).

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Comment by brett0000FF** [comment]
Removing beta banners entirely per Anna. 👍

File: content/en/opentelemetry/schema_semantics/host_metadata.md [hunk 1]
```diff
@@ -8,10 +8,6 @@ further_reading:
   text: "OpenTelemetry Support in Datadog"
 ---
 
-<div class="alert alert-info">
```
Comment (mx-psi) [inline content/en/opentelemetry/schema_semantics/host_metadata.md:11]: I don't think we should remove this one, as the original partner we had for testing this did not end up using the feature, so we have been able to do less validation than we intended

**Review by hestonhoffman — APPROVED** [review]

File: content/en/opentelemetry/schema_semantics/host_metadata.md (hunk 1 above)
Comment (brett0000FF) [reply content/en/opentelemetry/schema_semantics/host_metadata.md:11]: @mx-psi  Good to know! Given this, I will add the alert back but call it a "Preview" to align with our new product lifecycle. Let me know if you have any concerns about that approach!

Comment (mx-psi) [reply content/en/opentelemetry/schema_semantics/host_metadata.md:11]: That sounds good to me!

**Pull request merged**